### PR TITLE
#12 - Added abstract BaseActivity and added two static methods inside

### PR DESCRIPTION
**Pull request by jikolp95**
Added abstract BaseActivity and added two static methods inside FancyShowCaseView.

### CODE REVIEW

File: library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java [hunk 1]
```diff
@@ -219,6 +218,21 @@ public void onClick(View view) {
         }
     }
 
+
+    public static Boolean isVisible(Activity mActivity) {
```
Comment (faruktoptas) [inline library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java]: Can you add a comment to explain the methods please?

File: library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java [hunk 2]
```diff
@@ -219,6 +218,21 @@ public void onClick(View view) {
...
+        ViewGroup androidContent = (ViewGroup) mActivity.findViewById(android.R.id.content);
+        ViewGroup mRoot = (ViewGroup) androidContent.getParent().getParent();
+        FrameLayout mContainer = (FrameLayout) mRoot.findViewWithTag(CONTAINER_TAG);
+        return mContainer != null;
+    }
+
+    public static void hidePlease(Activity mActivity){
```
Comment (faruktoptas) [inline library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java]: Can you refactor the method name to `hideCurrent(Activity activity)` please?

File: library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java (hunk 1 above)
Comment (jikolp95) [inline library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java]: ok no problem

File: library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java (hunk 2 above)
Comment (jikolp95) [inline library/src/main/java/me/toptas/fancyshowcase/FancyShowCaseView.java]: no problem, one minute please

**Comment by faruktoptas** [comment]
Thank you for your contribution :)

**Pull request merged**